### PR TITLE
To Development - Update GET /recommendation to use Watson Natural Language API

### PR DESCRIPTION
**Pull request by MackHalliday**
### Description of Behavior

- Update GET `/recommendation` endpoint to use Watson Natural Language API, instead of MonkeyLearn.
- Error handling for if not enough characters for Watson sentiment analysis (Watson error 422). If there are less than 20 characters, then text sentiment is automatically neutral. 
-  Tests written with mock 

### Pull request checklist
Please check if your PR fulfills the following requirements:
- [x] Tests are written
- [x] Test coverage acceptable
- [X] Sad paths accounted for
- [X] Tested in Postman 

### Merge to
- [X] Development branch
- [ ] Master

### Additional Comments
Recommend see documentation for Watson Natural Language API: 

- [Documentation](http://watson-developer-cloud.github.io/node-sdk/v2.25.0/NaturalLanguageUnderstandingV1.html)
- [Example](https://github.com/watson-developer-cloud/python-sdk/blob/master/examples/natural_language_understanding_v1.py)

### CODE REVIEW

**Comment by MackHalliday** [comment]
env keys added to Heroku apps

**Comment by fentontaylor** [comment]
@MackHalliday This all looks great. Nice work on the testing as well. Can you change it to merge to dev branch instead? For some reason it won't let me make that change.

File: services/watson_service.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        if sentiment_value > 0.333:
+            return 1
+        elif sentiment_value < -0.333:
+            return 0
+        else:
+            return 0.5
+
+
+    def get_sentiment_value(self):
+        if len(self.text) > 20:
```
Comment (fentontaylor) [inline services/watson_service.py]: I understand what this code is doing, but not exactly why. Does Watson need at least 20 characters in order to perform the sentiment analysis?

Comment (MackHalliday) [reply services/watson_service.py]: Yes, Watson external API returns a 422 error if not enough characters for sentiment analysis.

Comment (fentontaylor) [reply services/watson_service.py]: 👍

**Pull request merged**